index.html

```diff
@@ -4,84 +4,117 @@
       rel="stylesheet"
     />
-    <link href="responsive.css" rel="stylesheet" />
-    <link href="style.css" rel="stylesheet" />
     <link
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <link href="style.css" rel="stylesheet" />
+    <link href="responsive.css" rel="stylesheet" />
   </head>
 
   <body>
     <div class="all-container">
       <header class="header">
         <div class="logo"><a href="#"> Marcelo's Dev </a></div>
-        <nav class="icon">
-          <a href="#"><img src="src/Icon-Menu.svg" alt="hamburger icon" /> </a>
+        <nav class="icon-nav">
+          <ul class="nav-items">
+            <li>Portfolio</li>
+            <li>about</li>
+            <li>Contact</li>
+            <li>
+              <a href="#"><img src="src/Icon-Mail.svg" alt="Mail Icon" /> </a>
+            </li>
+          </ul>
+          <a class="burgernav" href="mailto:marcelo.snajar@gmail.com"
+            ><img src="src/Icon-Menu.svg" alt="hamburger icon" />
+          </a>
         </nav>
       </header>
 
       <section class="section-2">
-        <h1>
-          Hey There.<br />
-          I'm Marcelo
-        </h1>
-        <h2>I’am a Software Developer</h2>
-        <p>
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hesitate to contact me.
-        </p>
+        <div class="section-container">
+          <h1>
+            Hey There.<br />
+            I'm Marcelo
+          </h1>
+          <h2>I’am a Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
         <div class="icon">
-          <a href="#"><img src="src/Social-Media.svg" alt="social-media" /> </a>
+          <ul>
+            <li>
+              <a href="#"><img src="src/github.svg" alt="Github logo" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="src/Linkedin.svg" alt="Linkedin logo" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="src/angel-list.svg" alt="Angel list logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="src/twitter.svg" alt="twitter logo" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="src/medium.svg" alt="medium logo" /></a>
+            </li>
+          </ul>
         </div>
       </section>
 
       <section class="my-works">
         <div class="works-section-a">
-          <h2>My Recent Works</h2>
+          <div class="recent">
+            <h2>My Recent Works</h2>
+            <div class="border-recent"></div>
```

Comment:
- [ ] The `works-section-a` is also not responsive and the elements overlap. It would be nice if you fix this. **_Hint:_** You can use `display: grid` to style the `<div class="wrapper">` and use `display: flex` to style the `<div class="works-section-a">` element 😉 

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144728477-23f0c146-701b-46a2-947a-2cf76cdb5bce.png)

## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144728491-318d5d55-25d5-4329-aa44-b79fc3d7e6c7.png)